### PR TITLE
Adding info on OEM debugging.

### PR DESCRIPTION
**Pull request by Sfshaza**
Staged:  https://sz-flutter.firebaseapp.com/oem-debugging/

Please review, @jacob314 @jwren .  iOS still to come.

### CODE REVIEW

File: src/oem-debugging/index.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+  <center><img src="images/debug-pane.png" style="border: 1px solid #ccc;"
+   alt="the debug pane showing that it's connected to the debugger and running the app"></center>
+
+  You can configure where the debug pane appears, or even tear it off to its own
+  window using the gear to the right in the Debug pane bar.
+  This is true for any inspector in Android Studio.
+
+* Add a breakpoint on the `counter++` line.
+
+* Hot reload the app.
```
Comment (jacob314) [inline src/oem-debugging/index.md]: A hot reload isn't needed for breakpoints to be active.

File: src/oem-debugging/index.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+
+Toggle display of the inspector using the vertical button to the
+right of the Android Studio window.
+
+  <center><img src="images/flutter-inspector.png" style="border: 1px solid #ccc;"
+   alt="screenshot showing the Flutter inspector"></center>
+
+## Flutter outline
+
+The Flutter Outline displays the current state of the widget tree.
```
Comment (jacob314) [inline src/oem-debugging/index.md]: It doesn't display the current state of the widget tree. It displays a tree visualization of the build method which may or may not be the same as the widget tree for the build method .

File: src/oem-debugging/index.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+
+{% comment %}
+TODO:
+Android Tips - How to assign a keyboard shortcut on the Mac?
+{% endcomment %}
+
+The rest of this guide shows how to set up your environment to debug OEM
+code. As you'd expect, the process works differently for iOS and Android.
+
+<aside class="alert alert-success" markdown="1">
```
Comment (jacob314) [inline src/oem-debugging/index.md]: is there some other dog where this pro tip should be included instead?

**Comment by chalin** [comment]
@Sfshaza - is this meant to replace #1210? If so, consider closing #1210.

Also note that the build is failing because of an invalid link:
```
./_site/oem-debugging/index.html
  *  internally linking to /performance/, which does not exist (line 571)
     <a href="/performance/">Performance Profiling</a>
```

**Comment by Sfshaza** [comment]
I've added this to the dash branch. Closing.

**Pull request closed without merging**